src/System.Drawing.Common/tests/ColorTranslatorTests.cs

```diff
@@ -188,7 +190,16 @@ public static IEnumerable<object[]> FromHtml_TestData()
         [MemberData(nameof(FromHtml_TestData))]
         public void FromHtml_String_ReturnsExpected(string htmlColor, Color expected)
         {
-            Assert.Equal(expected, ColorTranslator.FromHtml(htmlColor));
+            CultureInfo originalCulture = Thread.CurrentThread.CurrentCulture;
```

Comment:
I've done this a couple times before, but it's not incredibly common. So I'd say no, but it wouldn't really hurt anything either way.